File: server/router/admin/commerce_order.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/commerce_order"
	"server/global"
	"server/router/middleware"
)

func InitCommerceOrder(group *gin.RouterGroup) {
	g := group.Group("commerce/order", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("page", commerce_order.Page)
	g.POST("refund", commerce_order.Refund)
}

File: web/src/api/admin/commerce_order.js
import request from "../../request/index.js";
const baseUrl = '/v1/admin/commerce/order'
export const apiAdminCommerceOrderPage = (data) => request.request({url: `${baseUrl}/page`, method: 'POST', data})
export const apiAdminCommerceOrderRefund = (data) => request.request({url: `${baseUrl}/refund`, method: 'POST', data})
